### DISCUSSION TITLE
icon not working

### DISCUSSION THREAD

**Discussion by MrCroxx** [utilyre/barbecue.nvim#127] [Q&A]
Hi. Thanks for developing the cool plugin. I'm a newbie to neovim and trying to configure barbecue.nvim correctly.

I'm setting up barbecue with LazyVim in `barbecue.lua`:

```lua
return {
  {
    "utilyre/barbecue.nvim",
    name = "barbecue",
    version = "*",
    dependencies = {
      "SmiteshP/nvim-navic",
      "nvim-tree/nvim-web-devicons", -- optional dependency
    },
    opts = {
      -- configurations go here
    },
  },
}
```

But the icons are not displaying correctly.

<img width="479" alt="image" src="https://github.com/user-attachments/assets/7f896c9d-8e71-4e28-883c-1e519bc2774f">

May I ask what's going wrong in my env? Thank you.

**Comment by chianyungcode** [comment]
Same here
